src/hardware/mixer.cpp

```diff
@@ -260,6 +260,25 @@ void MixerChannel::Enable(const bool should_enable)
 	MIXER_UnlockAudioDevice();
 }
 
+void MixerChannel::ConfigureResampler()
+{
+	const auto in_rate = zoh_upsampler.enabled ? zoh_upsampler.target_freq
+	                                           : sample_rate;
+	const auto out_rate = mixer.freq;
+	if (in_rate == out_rate) {
+		resample = false;
+	} else {
+		if (!resampler) {
+			const auto num_channels = 2; // always stereo
+			const auto quality = 5;
+			resampler = speex_resampler_init(
+			        num_channels, in_rate, out_rate, quality, nullptr);
+		}
+		speex_resampler_set_rate(resampler, in_rate, out_rate);
+		resample = true;
+	}
+}
```

Comment:
Oh, what I mean by clobbering is that I remember we fixed up these values in the resampling PR - switching them to `constexpr`'s.  but these old changes are coming back in.

Reply:
Ahh yes, I see now that the rebase indeed happened because I'm seeing the new stuff in the visual diff here:

![2022-05-25_17-36](https://user-images.githubusercontent.com/1557255/170391640-8bad0c46-525c-4cdf-9306-fa416f382998.png)

Reply:
Ah okay, changing them back to `constexpr`.